Vagrantfile

```diff
@@ -0,0 +1,46 @@
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+if ! File.exists?('./oracle-xe-11.2.0-1.0.x86_64.rpm.zip')
+  puts 'Oracle XE database installation (oracle-xe-11.2.0-1.0.x86_64.rpm.zip) can not be found. Please download from Oracle homepage'
+  exit 1
+end
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  # Every Vagrant virtual environment requires a box to build off of.
+  config.vm.box = "chef/centos-6.6"
+
+  config.vm.hostname = "vagrant.oracle"
+
+  config.vm.network :forwarded_port, guest: 1521, host: 1521
+  # config.vm.network :forwarded_port, guest: 8080, host: 8080
+
+  config.vm.provider :virtualbox do |vb|
+    vb.name = "Ruby-PLSQL Oracle XE box"
+    # Speed up network
+    #   http://serverfault.com/a/453260/105586
+    #   http://serverfault.com/a/595010/105586
+    vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
+    vb.customize ["modifyvm", :id, "--natdnsproxy1", "on"]
+  end
+
+  config.vm.provision :shell, inline: "yum update -y"
+  config.vm.provision :shell, inline: "yum install -y libaio bc flex unzip"
+  config.vm.provision :shell, inline: "mkdir -p /opt/oracle"
+  config.vm.provision :shell, inline: "cp /vagrant/oracle-xe* /opt/oracle"
+  config.vm.provision :shell, inline: "cd /opt/oracle && unzip -q oracle-xe-11.2.0-1.0.x86_64.rpm.zip"
```

Comment:
What about in single step: `unzip -q -d /opt/oracle /vagrant/oracle/oracle-xe-11.2.0-1.0.x86_64.rpm.zip`?